draft-private-access-tokens.md

```diff
@@ -260,19 +260,20 @@ and corresponds to a unique ANON_ORIGIN_ID and a unique ANON_CLIENT_ID.
 It is assumed that Issuers make Oblivious HTTP configurations and policy verification
 keys available via the following API endpoints:
 
-- Access token policy verification key: /.well-known/access-token-key/policy=?
-- Access token policy window: /.well-known/access-token-window/policy=?
+- Issuer name public key (ISSUER_NAME_KEY): /.well-known/issuer-key
+- Access token policy window (ISSUER_POLICY_WINDOW): /.well-known/access-token-window
 
 The access token policy verification key is a struct of the following format:
 
 ~~~
 struct {
-   opaque public_key[Nk]; // Defined in [BLINDSIG]
-} AccessTokenKey;
+   opaque public_key[Nk]; // Defined in [OHTTP]
+} IssuerNameKey;
 ~~~
```

Comment:
We can probably just delete this struct now and use `KeyConfig` directly.

Reply:
@chris-wood updated, please check!